contrib/kafka/config/README.md

```diff
@@ -0,0 +1,87 @@
+# Apache Kafka Channels
+
+Kafka channels are those backed by [Apache Kafka](http://kafka.apache.org/) topics.
+
+## Deployment steps
+
+1. Setup [Knative Eventing](../../../DEVELOPMENT.md)
+1. If not done already, install an Apache Kafka cluster!
+
+   - For Kubernetes a simple installation is done using the
+     [Strimzi Kafka Operator](http://strimzi.io). Its installation
+     [guides](http://strimzi.io/quickstarts/) provide content for Kubernetes and
+     Openshift.
+
+   > Note: This _Channel_ is not limited to Apache Kafka installations on
+   > Kubernetes. It is also possible to use an off-cluster Apache Kafka
+   > installation.
+
+1. Now that Apache Kafka is installed, you need to configure the
+   `bootstrap_servers` value in the `config-kafka` ConfigMap,
+   located inside the `contrib/kafka/config/400-kafka-config.yaml` file:
+
+   ```yaml
+    apiVersion: v1
+    kind: ConfigMap
+    metadata:
+      name: config-kafka
+      namespace: knative-eventing
+    data:
+      # Broker URL. Replace this with the URLs for your kafka cluster,
+      # which is in the format of my-cluster-kafka-bootstrap.my-kafka-namespace:9092.
+      bootstrap_servers: REPLACE_WITH_CLUSTER_URL
+    ...
+   ``` 
+
+1. Apply the Kafka config:
+
+   ```
+   ko apply -f contrib/kafka/config
+   ```
+
+1. Create the kafka channel custom objects:
+
+   ```yaml
+    apiVersion: messaging.knative.dev/v1alpha1
+    kind: KafkaChannel
+    metadata:
+      name: my-kafka-channel
+    spec:
+      numPartitions: 1
+      replicationFactor: 3
+    ```
+    You can configure the number of partitions with `numPartitions`, as well as the replication factor with `replicationFactor`. If not set, both will default to `1`.
```

Comment:
should this say, how to connect to the broker ? or use as default channel ?

Reply:
I think we need better documentation for all channels. And we should put that in knative/docs. 
This is a copy/paster from the current provisioner-based README, plus some minor changes. 
IMO we can add that in a follow up in knative/docs. What do you think?